sql/modules/Drafts.sql

```diff
@@ -37,8 +38,8 @@ $$
                                    WHERE v.trans_id = gl.id)
             UNION
             SELECT id, transdate, invnumber as reference,
-                (SELECT name FROM eca__get_entity(entity_credit_account)),
-                invoice, amount_bc as amount, 'ap' as type
+                (SELECT name FROM eca__get_entity(entity_credit_account)) as vencusname,
```

Comment:
The code in `old/` uses `vcname` as the variable holding the "vendor or customer name". Our code doesn't really define a single vendor or customer concept. We have the company/person that you do business with (`entity`) or the "company in its role as customer/vendor" (which can be assigned a separate name - `entity_credit_account`). Since the `vencusname` will be filled with the `entity_credit_account.name`, I'm proposing we call the output column `eca_name` instead of `vencusname`.